tmux.conf

```diff
@@ -33,7 +31,8 @@ set -g status-right ''
 set -g history-limit 10000
 
 # switch to last pane
-bind-key C-a last-pane
+bind-key C-b last-pane
+bind-key C-s last-pane
```

Comment:
@tute I'm not aware of `prefix` standin.

With that said, I'm wondering if we might want to take a different approach with these keys. `last-pane` is already bound to `;` by default which is comfortably on the home row, and if using vim-tmux-navigator also bound to (unprefixed) `C-\`.

As an alternative, I typically map the prefix to `send-prefix` such that double pressing the prefix will send those keystrokes through. `C-b` for instance is `back-one-character` which can be useful, and `C-s` is `forward-incremental-history-search`. Without the `send-prefix` mapping you essentially have no way to send these keys.

If interested, the configuration would be:

``` tmux
bind-key C-b send-prefix
bind-key C-s send-prefix -2
```